### PR TITLE
Feature/sales force 002 smart login

### PR DESCRIPTION
**Pull request by OvidioMiranda**
I request review.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,6 +11,7 @@ sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
+    //jcenter()
```
Comment (CynthiaTerrazas) [inline build.gradle]: I think this line it is unnecessary.

File: src/main/java/org/fundacionjala/sfdc/CommonActions.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package org.fundacionjala.sfdc;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * Created by Simon on 07/08/2017.
+ */
+public final class CommonActions {
+
```
Comment (Gioserden) [inline src/main/java/org/fundacionjala/sfdc/CommonActions.java:10]: You're leaving a lot of space

**Comment by CynthiaTerrazas** [comment]
In the future maybe you could get the drivers are not in the source code, maybe with the help of some design pattern could improve this.

File: src/main/java/org/fundacionjala/sfdc/pages/Login.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        return setUserName(username).setPassword(password).clickLogin();
+    }
+
+    /**
+     * This Method make a login with other user to Salesforce application.
+     * @param userName User Name for Sales Force with other user.
+     * @param password Password for Sales Force with other user.
+     * @return Home page after login to Salesforce application.
+     */
+    public Home loginOtherUser(String userName, String password) {
```
Comment (Gioserden) [inline src/main/java/org/fundacionjala/sfdc/pages/Login.java]: You should apply strategy pattern here.

**Pull request merged**